internal/runners/platforms/remove.go

```diff
@@ -35,9 +35,12 @@ func (r *Remove) Run(ps RemoveRunParams) error {
 		return nil
 	}
 
-	return model.CommitPlatform(
-		r.Project,
-		model.OperationRemoved,
-		params.name, params.version, params.BitWidth,
-	)
+	err = requirements.ExecuteRequirementOperation(r.prime, params.name, params.version, params.BitWidth, model.OperationRemoved, model.NamespacePlatform)
+	if err != nil {
```

Comment:
Nit: Reusing err vars always triggers red flags for me. I don't recall the exact issue but I recall this can cause issues with asserting nil if the error types the var handles don't match.